### PR TITLE
code refactoring, and added forkenv, updated README

### PR DESCRIPTION
**Pull request by andrewssobral**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,52 @@
...
+import proactive
+
+print("Logging on proactive-server...")
+proactive_host = 'try.activeeon.com'
+proactive_port = '8080'
+proactive_url  = "http://"+proactive_host+":"+proactive_port
+print("Connecting on: " + proactive_url)
+gateway = proactive.ProActiveGateway(proactive_url)
+
+gateway.connect("bobot", "proactive")
```
Comment (marcocast) [inline README.md]: lets remove the user and password

Comment (amouhoub) [inline README.md]: @andrewssobral I think it is a very bad idea to hardcode these credentials here!
Maybe you can leave a comment and ask the user to use his paXXXX credentials obtained upon signing up.

**Review by marcocast — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (andrewssobral) [reply README.md]: fixed, thank you @amouhoub for the remark

Comment (andrewssobral) [reply README.md]: done!

**Review by paraita — APPROVED** [review]

**Pull request merged**